guides/source/routing.md

```diff
@@ -952,7 +952,7 @@ This would cause the routing to recognize paths such as:
 
 NOTE: The actual action names aren't changed by this option. The two paths shown would still route to the `new` and `edit` actions.
 
-TIP: If you find yourself wanting to change this option uniformly for all of your routes, you can use a scope.
+TIP: If you find yourself wanting to change this option uniformly for all of your routes, you can use a `scope`.
```

Comment:
Could you remove this please ? The sentence isn't referring to the method but the word.